src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from '../../../../common/Button/Button';
+import './CourseCard.css';
+import { getDuration } from '../../../../helpers/pipeDuration';
+import { dateGenerator } from '../../../../helpers/dateGenerator';
+
+export default function CourseCard(props) {
+	function Info(props) {
+		return (
+			<div className='info-item'>
+				<div className='info-title'>{props.infoTitle}</div>
+				<div className='info-content'>{props.infoContent}</div>
+			</div>
+		);
+	}
+
+	const courseInfo = [
+		['Authors: ', props.authors],
+		['Duration: ', `${getDuration(props.duration)} hours`],
+		['Created: ', dateGenerator(props.creationDate)],
+	];
+
+	return (
+		<section className='course-card' id={props.id}>
+			<div className='content'>
+				<h1 className='content-title'>{props.title}</h1>
+				<div className='content-description'>{props.description}</div>
+			</div>
+			<div className='info'>
+				<div className='info-list'>
+					{courseInfo.map(([title, content]) => (
+						<Info key={title} infoTitle={title} infoContent={content} />
```

Comment:
Could you please check this:
if all authors' names do not fit on one line, then the extra text should be cut off and '...' should be
added at the end of line

Reply:
Done!